docs/source/user-guide/compatibility.md

```diff
@@ -38,14 +38,6 @@ Comet uses the Rust regexp crate for evaluating regular expressions, and this ha
 regular expression engine. Comet will fall back to Spark for patterns that are known to produce different results, but
 this can be overridden by setting `spark.comet.regexp.allowIncompatible=true`.
 
-## Floating number comparison
```

Comment:
This change should be gone after rebasing and re-building?

Reply:
Thanks. This is gone now.